frontend/src/components/ChatBox/ChatBoxMessage/MessageBubble.tsx

```diff
@@ -54,12 +54,13 @@ function MessageBubble({
 				<b>Transformed: </b>
 			)}
 			{message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO && (
-				<p className="level-info-header">Information</p>
+				<>
+					<p className="level-info-header">Information</p>
+					<span className="visually-hidden"> message</span>
```

Comment:
Interesting! How does this work? Does a screen reader not read out the message twice here?

Reply:
Checked and it doesn't (at least not on windows narrator), so looks all good!

Reply:
I changed the message author 
this is just "message" as a string, not the {message}. 
That way, a screen reader will read: 
"Information message
Congratulations" You have completed this level. Please click on the next level to continue".

Reply:
ahh I see! Thanks for explaining